rct229/data_fns/table_G3_6_fns.py

```diff
@@ -0,0 +1,60 @@
+import rct229
+from rct229.data import data
+from rct229.data_fns.table_utils import find_osstd_table_entry
+from rct229.schema.config import ureg
+
+# This dictionary maps the ExteriorLightingAreas2019ASHRAE901TableG36 enumerations to
+# the corresponding lpd_space_type values in the file
+# ashrae_90_1_table_3_6.json
+
+building_exterior_enumeration_to_lpd_space_type_map = {
+    "UNCOVERED_PARKING_LOTS_AND_DRIVES": "Uncovered parking lots and drives",
+    "WALKWAY_NARROW": "Walkway - narrow",
+    "WALKWAY_WIDE": "Walkway - wide",
+    "PLAZA_AREAS": "Plaza Areas",
+    "SPECIAL_FEATURE_AREAS": "Special Feature Areas",
+    "STAIRWAYS": "Stairways",
+    "MAIN_ENTRANCE_DOOR": "Main entrance door",
+    "OTHER_ENTRANCE_OR_EXIT_DOORS": "Other entrance or exit doors",
+    "EXTERIOR_CANOPIES": "Exterior canopies",
+    "OUTDOOR_SALES_OPEN_AREAS": "Outdoor sales - open areas",
+    "STREET_FRONTAGE": "Street frontage",
+    "NON_TRADABLE_FACADE": "Non-tradable facade",
+}
+
+
+def table_G3_6_lookup(building_exterior_type_enum_val):
+    """Returns the lighting power density for a building_exterior as
+    required by ASHRAE 90.1 Table G3.6
+    Parameters
+    ----------
+    building_exterior_type : str
+        One of the ExteriorLightingAreas2019ASHRAE901TableG36 enumeration values
+
+    Returns
+    -------
+    dict
+        { lpd: Quantity - The lighting power density in watt per square foot given by Table G3.6, linear_lpd: Quantity - The lighting power density in watt per linear foot given by Table G3.6 }
```

Comment:
Black won't reformat inside a comment, so you should break this into multiple lines.